aten/src/ATen/native/native_functions.yaml

```diff
@@ -1517,6 +1517,12 @@
   variants: function, method
   device_guard: False
 
+- func: mkldnn_reshape(Tensor self, int[] shape) -> Tensor
+  device_guard: False
+  requires_tensor: True
```

Comment:
you don't need `requires_tensor` if you have `dispatch`

Reply:
yep I know this, just for documentation purpose :-)